src/object-attribute.scss

```diff
@@ -0,0 +1,47 @@
+@import "./new-settings";
+@import "./mixins";
+
+$fd-object-attribute-text-decoration-none: none;
+$fd-object-attribute-text-decoration-underline: underline;
+
+$block: #{$fd-namespace}-object-attribute;
+
+.#{$block} {
+  @include fd-reset();
+  @include fd-ellipsis();
+
+  color: var(--sapContent_LabelColor);
+  display: block;
+
+  &--link {
+    @include action-cursor();
+
+    text-decoration: $fd-object-attribute-text-decoration-none;
+    color: var(--sapLinkColor);
+
+    &:visited {
+      text-decoration: $fd-object-attribute-text-decoration-none;
+      color: var(--sapLink_Visited_Color);
+    }
+
+    &:hover {
+      @include fd-hover() {
+        text-decoration: $fd-object-attribute-text-decoration-underline;
+        color: var(--sapLink_Hover_Color);
+      }
+    }
+
+    &[disabled],
+    &[aria-disabled="true"] {
```

Comment:
there's fd-disabled mixin you canuse

Reply:
fd-disabled is causing build failures, so I removed it.